PVC/Mesh_voxelisation/READ_stl.m

```diff
@@ -0,0 +1,274 @@
+function [coordVERTICES,varargout] = READ_stl(stlFILENAME,varargin)
+% READ_stlascii  Read mesh data in the form of an <*.stl> file
+%==========================================================================
+% FILENAME:          READ_stl.m
+% AUTHOR:            Adam H. Aitkenhead

```

Comment:
@gmangeat did you check if we have the permission to add in our repos with MIT license?